nautobot_circuit_maintenance/handle_notifications/sources.py

```diff
@@ -253,8 +254,10 @@ def extract_provider_data_types(email_source: str) -> Tuple[str, str, str]:
             )
         """
         for provider in Provider.objects.all():
-            if "emails_circuit_maintenances" in provider.custom_field_data:
-                if email_source in provider.custom_field_data["emails_circuit_maintenances"].split(","):
+            if provider.custom_field_data.get("emails_circuit_maintenances"):
```

Comment:
Why not in single if conditional?

Reply:
You mean:

```python
             if (
                 provider.custom_field_data.get("emails_circuit_maintenances")
                 and email_source in [
                    source.strip() for source in provider.custom_field_data["emails_circuit_maintenances"].split(",")
                ]
            ):
```

Feels a bit unwieldy to me but I'm not opposed to it.

Reply:
i won't lose sleep over it and only see pros/cons. If I see dual if statements I am confused, conversely if I see complex if statement its hard to read.

Also thinking to invert the truth table to keep it less nested.

```
            if provider.custom_field_data.get("emails_circuit_maintenances") is None:
               continue
            if email_source in [
                    source.strip() for source in provider.custom_field_data["emails_circuit_maintenances"].split(",")
                ]:
                provider_type = provider.slug
                break
```